src/einspline/tests/test_one.cpp

```diff
@@ -182,35 +178,36 @@ TEST_CASE("multi_cuda_wrapper", "[einspline]")
   BCtype_s xBC;
   xBC.lCode = NATURAL;
   xBC.rCode = NATURAL;
-  multi_UBspline_1d_s* s = create_multi_UBspline_1d_s(x_grid, xBC, 1);
+  auto s    = std::unique_ptr<multi_USspline_1d_s, void (*)(void*)>{create_multi_UBspline_1d_s(x_grid, xBC, 1),
```

Comment:
UBspline caught by CI.

Reply:
Fixed. Surprised clang 11 built with the wrong pointer type.

Reply:
It is under CUDA build.